openshift/buildConfig-template.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: graph-backup-buildconfig
+  annotations:
+    description: >
+      This is Thoth Graph Backup CronJob BuildConfig, this template is meant to be used by Bots,
+      but could also be used by humans...
+    openshift.io/display-name: "Thoth: Graph Backup Job BuildConfig"
+    version: 0.1.0
+    tags: thoth,graph-backup,ai-stacks,aistacks,graph
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to deploy Thoth Graph Backup Job BuildConfig to OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    template: graph-backup-buildconfig
+    app: thoth
+    component: configmap
+
+objects:
+  - kind: BuildConfig
+    apiVersion: build.openshift.io/v1
+    metadata:
+      name: graph-backup-job
+      annotations:
+        thoth-station.ninja/template-version: 0.1.0
+      labels:
+        app: thoth
+        component: graph-backup
+    spec:
+      resources:
+        limits:
+          cpu: 1
+          memory: 768Mi
+        requests:
+          cpu: 1
+          memory: 768Mi
+      output:
+        to:
+          kind: ImageStreamTag
+          name: "graph-backup-job:${IMAGE_STREAM_TAG}"
+      source:
+        type: Git
+        git:
+          uri: ${GITHUB_URL}
+          ref: ${GITHUB_REF}
+      strategy:
+        type: Source
+        sourceStrategy:
+          from:
+            kind: ImageStreamTag
+            name: s2i-thoth-ubi8-py36:latest
```

Comment:
To install Postgresql into the image, .s2i/bin/assemble is added to the PR.  However, due to subscription manager and non-root user it doesn't execute. 
Should I proceed with the different images using the same s2i method, or move to dockerfile based method?

Reply:
As the only thing added is postgresql-96, I think it would be good idea to create a docker build that just extends `s2i-thoth-ubi8-py36:latest`, something like:

```dockerfile
FROM s2i-thoth-ubi8-py36:latest

USER 0
RUN yum install -y postgresql-96
USER 1001
```

and pushes the resulting image to `s2i-thoth-ubi8-py36-postgres:latest` (or something like that). The s2i build can then use extended `s2i-thoth-ubi8-py36-postgres:latest`.

Reply:
Thanks @fridex for the feedback, Always good to know the correct way to do things :smile: .
I will make the suitable changes.